Sources/Sentry/include/SentrySpan.h

```diff
@@ -30,6 +30,11 @@ SENTRY_NO_INIT
  */
 @property (nullable, readonly) NSDictionary<NSString *, id> *data;
 
+/**
+ * A key-value pairs holding additional data about the span.
+ */
+@property (nullable, readonly) NSDictionary<NSString *, id> *tags;
```

Comment:
`m`: As the `SentrySpan` class implements the `SentrySpan` protocol, I think we don't have to declare this and `setTagValue` here right? Same applies for `SentryTracer`.